src/pepr/policies/security.ts

```diff
@@ -25,7 +25,24 @@ import { exemptionAnnotationPrefix, isExempt, markExemption } from "./exemptions
  */
 When(a.Pod)
   .IsCreatedOrUpdated()
-  .Mutate(markExemption(Policy.DisallowPrivileged))
+  .Mutate(request => {
+    markExemption(Policy.DisallowPrivileged)(request);
+    if (request.HasAnnotation(`${exemptionAnnotationPrefix}.${Policy.DisallowPrivileged}`)) {
+      return;
+    }
+
+    // Check if any containers defined in the pod do not have the `allowPrivilegeEscalation` field present. If not, include it and set to false.
+    for (const container of containers(request)) {
+      container.securityContext = container.securityContext || {};
+      if (
+        container.securityContext.allowPrivilegeEscalation === undefined &&
+        !container.securityContext.privileged
```

Comment:
Did you happen to check the case where `CAP_SYS_ADMIN` is added as a capability? Re-reading the [docs here](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) it seems like that might fall in a similar bucket of "requested privileges elsewhere" and maybe should opt-out of this mutation rather than requiring the end user to do it themselves?

Assuming we do the validate mentioned in my comment below the end user would likely need to have an exemption for this case anyways so this might reduce the things they need to add/do by one.

Reply:
That's a good callout. I'll add a check to see if the `CAP_SYS_ADMIN` capability is included alongside the existing check for `privileged`.